### PR TITLE
fix legend widget object title

### PR DESCRIPTION
**Pull request by gitcontributor**
## Description

**Please check if the PR fulfills these requirements**
- [ x ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ x ] Bugfix

## Issue

**What is the current behavior?**

#5425 

**What is the new behavior?**

If object comes as a title property, gets the property of its object.
## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by gitcontributor** [comment]
This issue is related to C028 issue https://github.com/geosolutions-it/MapStore2-C028/issues/153.
This bug appears again, because in my previous PR i fixed widgets that ware created from the CompactCatalog component. But the legend widget is a descendant of LegendWizard, not CompactCatalog.

**Comment by tdipisa** [comment]
This PR need to be backported on both 2020.01.xx and 2020.02.xx

File: web/client/components/widgets/builder/wizard/LegendWizard.jsx [hunk 1]
```diff
@@ -19,6 +21,14 @@ const Wizard = wizardHandlers(require('../../../misc/wizard/WizardContainer'));
 const StepHeader = require('../../../misc/wizard/StepHeader');
 const Message = require('../../../I18N/Message');
 const emptyLegendState = require('../../enhancers/emptyLegendState');
+const {currentLocaleSelector} = require('../../../../selectors/locale');
+
+const localeSelector = createSelector(
```
Comment (offtherailz) [inline web/client/components/widgets/builder/wizard/LegendWizard.jsx:26]: Could you please connect the state at plugin level (`plugins/widgetBuilder/LegendBuilder.jsx`).
We use to connect component only at plugin level (we will define better this policy in next tech debt

Comment (gitcontributor) [reply web/client/components/widgets/builder/wizard/LegendWizard.jsx]: done

File: web/client/components/widgets/builder/wizard/LegendWizard.jsx [hunk 2]
```diff
@@ -29,15 +28,16 @@ const enhancePreview = compose(
             description: <Message msgId="widgets.builder.errors.noMapAvailableForLegendDescription" />
         }
     ),
-    emptyLegendState(false)
+    emptyLegendState(false),
```
Comment (MV88) [inline web/client/components/widgets/builder/wizard/LegendWizard.jsx:31]:
```suggestion
    emptyLegendState(false)
```

Comment (MV88) [reply web/client/components/widgets/builder/wizard/LegendWizard.jsx:31]: i think this needs to be removed aswell

Comment (gitcontributor) [reply web/client/components/widgets/builder/wizard/LegendWizard.jsx]: done

**Comment by offtherailz** [comment]
Great :+1: 
Please backport on both 2020.01.xx and 2020.02.xx

**Pull request merged**